### PR TITLE
FIX: eddy current sweep Maxwell

### PR DESCRIPTION
**Pull request by gmalinve**
## Description
**Please provide a brief description of the changes made in this pull request.**

## Issue linked
**Please mention the issue number or describe the problem this pull request addresses.**

## Checklist
- [ ] I have tested my changes locally.
- [ ] I have added necessary documentation or updated existing documentation.
- [ ] I have followed the coding style guidelines of this project.
- [ ] I have added appropriate tests (unit, integration, system).
- [ ] I have reviewed my changes before submitting this pull request.
- [ ] I have linked the issue or issues that are solved by the PR if any.
- [ ] I have agreed with the Contributor License Agreement ([CLA](https://developer.ansys.com/form/cla-acceptance)).

### CODE REVIEW

File: tests/system/general/test_28_Maxwell3D.py [hunk 1]
```diff
@@ -234,26 +234,26 @@ def test_06_eddycurrent(self):
...
-        assert Setup.props["SaveAllFields"]
-        assert Setup.add_eddy_current_sweep("LinearCount", dc_freq, stop_freq, count, clear=False)
-        assert isinstance(Setup.props["SweepRanges"]["Subrange"], list)
-
-        assert Setup.update()
-        assert Setup.enable_expression_cache(["CoreLoss"], "Fields", "Phase='0deg' ", True)
-        assert Setup.props["UseCacheFor"] == ["Pass", "Freq"]
-        assert Setup.disable()
-        assert Setup.enable()
+        assert setup.add_eddy_current_sweep("LinearStep", dc_freq, stop_freq, count, clear=True)
```
Comment (jvela018) [inline tests/system/general/test_28_Maxwell3D.py:246]: Here I'm just being pedantic but maybe we can change the variable name from dc_freq to initial_freq, and count to freq_inc (for increment..or something along those lines). Note that the value of the increment is in Hz so the value count reads odd to me 

image

**Review by jvela018 — CHANGES_REQUESTED** [review]
@gmalinve , I added some minor comments. Let me know if they make any sense.

File: tests/system/general/test_28_Maxwell3D.py (hunk 1 above)
Comment (Samuelopez-ansys) [reply tests/system/general/test_28_Maxwell3D.py:246]: Even better, to have more consistency with HFSS.

start_frequency
stop_frequency

And sweep_type 
And step_size

Comment (gmalinve) [reply tests/system/general/test_28_Maxwell3D.py:246]: @jvela018 @Samuelopez-ansys agree, I will change this in the function handler

File: src/ansys/aedt/core/modules/solve_setup.py [hunk 2]
```diff
@@ -181,10 +181,7 @@ def _init_props(self, is_new_setup=False):
                     setups_data = self.p_app.design_properties["AnalysisSetup"]["SolveSetups"]
                     if self.name in setups_data:
                         setup_data = setups_data[self.name]
-                        if "Sweeps" in setup_data and self.setuptype not in [
-                            0,
-                            7,
-                        ]:  # 0 and 7 represent setup HFSSDrivenAuto
+                        if "Sweeps" in setup_data and self.setuptype != 0:  # 0 and 7 represent setup HFSSDrivenAuto
```
[the remark this answers is not in the document]
Comment (gmalinve) [reply src/ansys/aedt/core/modules/solve_setup.py:184]: for EddyCurrent setup type the "Sweep" is never present in setup_data.

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

**Review by jvela018 — APPROVED** [review]
Looks good to me.

**Pull request merged**